packages/xstate-wallet/src/store/channel-store-entry.ts

```diff
@@ -255,7 +255,7 @@ export class ChannelStoreEntry {
     }
 
     const {signedStates} = this;
-    const turnNums = _.map(signedStates, s => s.turnNum.toHexString());
+    const turnNums = _.map(signedStates, s => parseInt(s.turnNum.toHexString(), 16));
```

Comment:
Will this work with proper `BigNumber`s? While we're using `BigNumber` for `turnNum` (I think we should _stop_ doing that) perhaps we should just set `isReverseSorted` to accept `BigNumber[]`